### PR TITLE
[WIP] generalizing script for class

### PR DESCRIPTION
**Pull request by jdkent**
Changes proposed in this pull request
- Removing fieldmap processing (this will need some TLC to generalize and I don't feel we have the time to tackle this issue)
- Generate the T1 mask within the script (so we only rely on what's inside the BIDS directory and don't pull from derivatives)
- Adapt the wrapper to work with and without sessions

I'm still in the middle of making changes, but I wanted to lay out conceptually what I'm trying to accomplish.

### CODE REVIEW

**Comment by mwvoss** [comment]
I agree with all these proposed changes

**Comment by mwvoss** [comment]
note for later as well, @tbweng will have input on the many things that look to be hard-coded into the run_getroicorrs.m script. This script has gone through a lot of changes based on needs of Matt Sutterer and Tim. Let's try to get less hard-coded in there, including very importantly the numvols.

@tbweng do you know where the getroicorrs.m script went in master?

File: processRestingState_bids_wrapper.sh [hunk 1]
```diff
@@ -198,63 +176,30 @@ else
...
+  --regmode=6dof
+
+clobber "${rsOut}"/preproc/nonfiltered_smooth_data.nii.gz &&\
+"${scriptdir}"/restingStatePreprocess.sh \
+  --epi="${rsOut}"/mcImg_stripped.nii.gz \
+  --t1brain="${T1_RPI_brain}" \
+  --tr=2 \
+  --te=30 \
+  --smooth=6 \
+  "${aromaArg}"
```
Comment (tbweng) [inline processRestingState_bids_wrapper.sh]: Are you hoping to remove fieldmap processing entirely (rather than having flag)?

**Comment by tbweng** [comment]
I'm not sure if getroicorrs.m ever made it in the initial commit. Would you like me to add my working version to the master as a starting point? 

Another functionality that I've been trying to implement in getroicorrs.m is parfor loops a la firstlevelseeding_parallel.m. It would greatly improve its speed especially with a large sublist and/or large seedlist. I have not been able to successfully implement.

File: processRestingState_bids_wrapper.sh (hunk 1 above)
Comment (mwvoss) [reply processRestingState_bids_wrapper.sh]: a flag seems fine to keep for the main scripts; though we don't need functionality for the fieldmaps for the class version

**Comment by mwvoss** [comment]
@tbweng sure adding your working version is a good start

**Comment by jdkent** [comment]
Going to close this for now.

**Pull request closed without merging**